src/components/Expandable/Expandable.tsx

```diff
@@ -1,19 +1,23 @@
 import React, { useCallback, useState } from 'react';
 import classNames from 'classnames';
+import ChevronDownIcon from '@moda/icons/chevron-down-12';
+import ChevronUpIcon from '@moda/icons/chevron-up-12';
 import { Clickable } from '../Clickable';
 import './Expandable.scss';
 
 export type ExpandableProps = React.HTMLAttributes<HTMLDivElement> & {
   name: string;
   expanded?: boolean;
   children: React.ReactNode;
+  icon?: 'chevron' | 'default';
```

Comment:
There's no explicit indication of what "default" is.  Can the default say "plus-minus"?